.github/FUNDING.yml

```diff
@@ -1,3 +0,0 @@
-# These are supported funding model platforms
-
-open_collective: pylabel
```

Comment:
We should keep this file. It should not cause a problem.

Reply:
Fixed as suggested in the [commit](https://github.com/pylabel-project/pylabel/pull/8/commits/9470c31e081b77fb3c07cd752b91900b14cb017f)